ch5/Makefile

```
@@ -15,10 +15,10 @@ crud-pydantic:
 	uvicorn crud_pydantic:app --reload
 
 server:
-	docker build -f Dockerfile -t ch5-api-server .
-	docker run --name api-server -p 8000:8000 -d ch5-api-server
+	docker build -f Dockerfile -t fast-api-server .
+	docker run --name api-server -p 8000:8000 -d fast-api-server
```

ch5 는 독립적인 챕터라서 ch5 로 그냥 놔두는 것이 좋을 것 같습니다.
챕터명이 없으면 docker compose 로 묶이는 다른 것들과 헷갈릴 우려가 있습니다.
```suggestion
	docker build -f Dockerfile -t ch5-api-server .
	docker run --name api-server -p 8000:8000 -d ch5-api-server
```

---

잊을 뻔 했네요! 반영했습니다~!